k8up.rst

```diff
@@ -0,0 +1,26 @@
+K8up (Backup Service)
+=====================
+
+.. admonition:: Beta Warning
+    :class: note
+
+    This service is in productive use by some of our customers.
+    It is still in beta, though. Features may be added or removed.
+    We're happy about your feedback!
+
+What is K8up?
+-------------
+
+On APPUiO we've made backing up your data simple. All you need to do is
+create a backup ``Schedule`` object in your APPUiO namespace. This will
+backup the file system data of your PVCs in that namespace.
+
+- Example: `Backup Schedule <https://k8up.io/docs/latest/backup.html>`__ (k8up documentation)
```

Comment:
I would just remove this line. We already have a link to K8up documentation, no need to maintain a link to a specific page in the docs that might change in any K8up release.